templates/angular/igx-ts/combo/default/index.ts

```diff
@@ -0,0 +1,20 @@
+import { IgniteUIForAngularTemplate } from "../../../../../lib/templates/IgniteUIForAngularTemplate";
+
+class IgxComboTemplate extends IgniteUIForAngularTemplate {
+	constructor() {
+		super(__dirname);
+		this.components = ["Combo"];
+		this.controlGroup = "Grids & Lists";
+		this.listInComponentTemplates = true;
+		this.id = "combo";
+		this.projectType = "igx-ts";
+		this.name = "Combo";
+		this.description = "Basic IgxCombo with templating";
+		this.dependencies = [{
+			from: "igniteui-angular",
+			import: ["IgxComboModule", "IgxDropDownModule", "IgxCheckboxModule", "IgxToggleModule",
```

Comment:
Those imports seem a bit excessive - I don't see `IgxIcon` or button being used as stand-alone cpmponenets, just the `<igx-combo>` and some divs. So it should be enough to import just `IgxComboModule` combo, right?